### PR TITLE
remove the stable/ target altogether - temporary

### PR DESCRIPTION
**Pull request by lochel**
- additionally copy stable releases to
  https://build.openmodelica.org/omsimulator/stable/

### CODE REVIEW

**Comment by lochel** [comment]
@AnHeuermann maybe it's even better to always upload to the nightly/ directory and additionally to stable/ in case of releases. Might be less confusing and would make it easier for the pip package to find artifacts.

**Comment by AnHeuermann** [comment]
It's a lot of duplicate code, but I don't see how we can simplify that.

File: Jenkinsfile [hunk 1]
```diff
@@ -678,6 +678,58 @@ EXIT /b 1
           }
         }
 
+        stage('upload-artifacts-stable') {
```
Comment (AnHeuermann) [inline Jenkinsfile:681]: Stage  `upload-artifacts` will also upload to stable/

Comment (lochel) [reply Jenkinsfile:681]: Not with the changes from this pr, right?

**Comment by AnHeuermann** [comment]
Also we don't need to upload the same files twice. Can't we just create a link from nightly/ to stable (or the other way around)?

**Comment by lochel** [comment]
> Also we don't need to upload the same files twice. Can't we just create a link from nightly/ to stable (or the other way around)?

I guess so, yes. However, it just doesn't matter since we have so few releases and they are very small in size.

File: Jenkinsfile (hunk 1 above)
Comment (AnHeuermann) [reply Jenkinsfile:681]: Oh yes. If `params.STABLE` is true this stage will rune and upload to `stable/`.
The stage above will use `DEPLOYMENT_PREFIX = getDeploymentPrefix()` and get `nightly/`

**Comment by AnHeuermann** [comment]
Maybe I should check how big the folder at https://libraries.openmodelica.org/fmi-crosschecking/ has grown. Maybe we need some housekeeping as well.

**Comment by AnHeuermann** [comment]
> Or we just remove the stable/ target altogether

That's fine with me.

**Pull request merged**